include/dirac_quda.h

```diff
@@ -230,6 +230,16 @@ namespace quda {
 			    const QudaParity parity, const ColorSpinorField &x,
 			    const double &k) const = 0;
 
+    /**
+       @brief Similar to the Xpay version of Dslash, but used only by the Laplace op for 
+       smearing.
+    */    
+    virtual void SmearOp(ColorSpinorField &, const ColorSpinorField &, 
```

Comment:
It's an additional method, since requires a two-link field in addition to the regular staggered field. It's also optional, while dslash Xpay is mandatory.  (It is similar in terms of argument list , not the implementation, sorry for the confusion)

Reply:
Ah thanks for the clarification @alexstrel.

Reply:
Please also clarify this in the doxygen comment.